app/models/concerns/orchestration/compute.rb

```diff
@@ -102,6 +103,7 @@ def setUserData
 
     return false if errors.any?
     logger.info "Revoked old certificates and enabled autosign for UserData"
+    return true
```

Comment:
Why would implicit return not work here?  Something like this is more clear to me, and more helpful logs:

```ruby
if errors.any?
  logger.info "Error revoking old certificates and enabled autosign for UserData: #{errors.full_messages.to_sentence}"
  false
else     
  logger.info "Revoked old certificates and enabled autosign for UserData"
  true
end
```

```